src/store/group-management/index.ts

```diff
@@ -63,11 +75,30 @@ const slice = createSlice({
     setAddMemberError: (state, action: PayloadAction<GroupManagementState['addMemberError']>) => {
       state.addMemberError = action.payload;
     },
+    setEditConversationErrors: (
```

Comment:
This seems like it's doing more than `set` the errors. It's adding errors?

Is there actually a reason to maintain the old errors? If we're setting them when something fails that probably means the previous error is no longer valid?

Reply:
I..was keeping the errors as separate `{ image: <error_msg>, general: <error_msg>}`. This is what we're adding/setting in the sagas. NO need to maintain these errors separately, but..i thought that would make more sense.. (to make errors look more verbose)? 

Why do i feel i actually did the opposite of that here 😅